packages/labltk/labltk.8.06.6/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "http://labltk.forge.ocamlcore.org/"
+bug-reports: "https://github.com/garrigue/labltk/issues"
+dev-repo: "git+https://github.com/garrigue/labltk.git"
+license: "LGPL with linking exception"
+patches: [ "auxlibs-in-META.patch" ]
+build: [
+  ["./configure" "-use-findlib" "-installbindir" bin]
+  [make "library" "opt"]
+]
+install: [
+  [make "install"]
+]
+remove: [["ocamlfind" "remove" "labltk"]]
+depends: [
+  "ocaml" {>= "4.07"}
+  "ocamlfind"
```

Comment:
both of these packages don't seem to use ocamlfind as a library or use it at runtime. Could you add the `{build}` attribute to them?

Reply:
Sure. Done.